handlers/authentication.go

```diff
@@ -210,6 +211,62 @@ func performOpenshiftAuthentication(w http.ResponseWriter, r *http.Request) bool
 	return true
 }
 
+func performTokenAuthentication(w http.ResponseWriter, r *http.Request) bool {
+	err := r.ParseForm()
+
+	if err != nil {
+		RespondWithJSONIndent(w, http.StatusInternalServerError, fmt.Errorf("error parsing form info: %+v", err))
```

Comment:
minor, here and in other places you're not returning a JSON string, are you?  I'd suggest:

`RespondWithDetailedError(w, http.StatusInternalServerError, "Error parsing form info", err.Error())`

Reply:
Ops, yes! I fixed it.
This came from the old PR (https://github.com/kiali/kiali/pull/1216) when the detailed error function didn't exist yet.